stores/templates/stores/dashboard/store_list.html

```diff
@@ -64,30 +64,30 @@ <h2>{{ queryset_description }}</h2>
                 {% for store in store_list %}
                     <tr>
                         <th><a href="{% url 'stores-dashboard:store-update' store.pk %}">{{ store.name }}</a></th>
-                        <td>{{ store.address.street }}</td>
+                        <td>{{ store.address.street|linebreaks }}</td>
```

Comment:
I think `linebreaksbr` may be preferable here, since we are not wrapping the other cell values in `<p>` tags.

Reply:
Fixed.